LibZipSharp.UnitTest/ZipTests.cs

```diff
@@ -204,7 +204,11 @@ public void EnumerateSkipDeletedEntries (bool deleteFromExistingFile)
 				ValidateEnumeratedEntries (zip);
 			}
 			finally {
-				zip?.Dispose ();
+				try {
+					zip?.Dispose ();
+				} catch {
```

Comment:
@grendello @BretJohnson I'm not sure what to do about this. If we downgrade to 1.5.1 this test failed with an exception, which doesn't happen on the newer versions. 

Should we try to build the newer version on Mac like we did with 1.5.1 ?

Reply:
This might be the only option...